### DISCUSSION TITLE
Multiple programs that share the same entry in Apps & Features

### DISCUSSION THREAD

**Discussion by marco-eckstein** [microsoft/winget-pkgs#132199] [Q&A]
There is a publisher that offers separate installers for multiple programs. However, in Apps & Features they all share the same entry for the uninstaller. After starting the uninstaller, you can select which programs to uninstall.

I had already submitted a package for one of these programs (accepted as package `Line6.Line6Monkey`). Now I see that after installing the other programs manually and uninstalling the WinGet package with `winget uninstall`, `winget list` still lists the package as installed  - because the uninstaller entry in Apps & Features is still there for the other programs.

Is it possible to manage these programs with WinGet at all? If so, how? Or should I retract the package `Line6.Line6Monkey`?

**Comment by Trenly** [comment]
If the programs share the same uninstall entry, winget will not be able to differentiate them. Each app should have its own entry in Add/Remove Programs. 

There are some tools in the newer manifests to help with this, but they are not yet fully documented or supported
